gateway/1.8/modules/ROOT/pages/_partials/task-config-readiness-liveness.adoc

```diff
@@ -0,0 +1,191 @@
+| `--allow-envoy-errors`| The flag to allow Envoy configuration errors
+| `--allow-policy-errors`| The flag to allow policy configuration errors
+| `initialDelaySeconds` | The time in seconds to wait after startup before executing the first readiness probe
+| `periodSeconds` | The period in seconds between each readiness probe
+| `failureThreshold` | The number of failed readiness probes before the Kubernetes readiness status is set to `false`
+| `timeoutSeconds` | The time in seconds before the readiness probe times out
+|===
++
+To modify the default parameters, you must update your Helm chart. You can modify these parameters during or after the initial installation of the Helm chart. For more information about how to update a Helm chart, see xref:flex-gateway-k8-change-helm-settings.adoc[].
++
+NOTE: When using a readiness probe in Kubernetes (or Openshift) and Connected Mode, omit the `--wait` flag in the Helm chart used to deploy Flex Gateway. Not doing so might result in an installation failure because Helm waits for a readiness state of `true`. If there is no API deployed to your Flex Gateway instance, the readiness state is `false`. This is not an issue with custom readiness probe configurations or gateways that have an API configured.
++
+[discrete]
+==== Configure External Components
++
+You configure your external components to query the readiness state. For example, in AWS virtual machines that run behind a load balancer, configure the readiness query as a health check. To expose the readiness API to external components, apply the following configuration to the Flex Gateway instance:
++
+[source,yaml]
+----
+apiVersion: gateway.mulesoft.com/v1alpha1
+kind: Configuration
+metadata:
+ name: probe
+spec:
+ probe:
+   enabled: true
+   port: 3000
+----
++
+Applying the YAML configuration snippet enables the readiness API to respond to `GET` requests:
++
+`\http://<flex-gateway-url>:<port-configured>/probes/readiness?allowEnvoyErrors=<true|false>&allowAPIErrors=<true|false>&allowPolicyErrors=<true|false>`
++
+[discrete]
+==== Configure Error Tolerance
++
+The default Kubernetes readiness probe is configured to be error-tolerant. If a configuration error occurs (for example, if you misconfigure a policy), the readiness state still changes to `true` once the API instances and policies are deployed. This enables Flex Gateway to scale even though there might be configuration errors.
++
+You can also configure the readiness probe to be error-intolerant, meaning that misconfigured gateway instances never reach a readiness state of `true`.
++
+To configure the Helm chart for error intolerance, remove the following flags from the readiness probe: `--allow-api-errors`, `--allow-envoy-errors`, and `--allow-policy-errors`. In deployments that query the readiness state through HTTP requests, use query parameters: `allowEnvoyErrors=false`, `allowAPIErrors=false`, and `allowPolicyErrors=false`.
```

Comment:
```suggestion
To configure the Helm chart for error intolerance, remove the following flags from the readiness probe: 

* `--allow-api-errors`
* `--allow-envoy-errors`
* `--allow-policy-errors`. 

In deployments that query the readiness state through HTTP requests, use these query parameters: 

* `allowEnvoyErrors=false`
* `allowAPIErrors=false`
* `allowPolicyErrors=false`.
```
I think this change would help visualize the number of flags/parameters to change. Please check in a local build and apply this change if you consider it is ok.

Reply:
Fixed! Thanks.